app/_src/gateway/reference/cli.md

```diff
@@ -435,13 +442,22 @@ Options:
 
 ---
 
+{% if_version gte:3.10.x %}
+### kong runner
+{% else %}
 ### kong runner
 {:.badge .enterprise}
+{% endif_version %}
+
 
 ```
+{% if_version gte:3.10.x %}
+Usage: kong runner file.lua [args]
+{% else %}
 Usage: kong runner [file] [args]
+{% endif_version %}
```

Comment:
```suggestion
Usage: kong runner file.lua [args]
```
This just looks like a clarification fix, doesn't seem like it needs to be version-tagged.